deploy/docker-compose/.env

```diff
@@ -0,0 +1,85 @@
+# Container Images
+INFORMATICS_GATEWAY_IMAGE=ghcr.io/project-monai/monai-deploy-informatics-gateway:0.3.0
+INFORMATICS_GATEWAY_CONFIG_IMAGE=dwdraju/alpine-curl-jq:latest
+WORKFLOW_MANAGER_IMAGE=ghcr.io/project-monai/monai-deploy-workflow-manager:develop-nightly-2022-09-12
+TASK_MANAGER_IMAGE=ghcr.io/project-monai/monai-deploy-task-manager:develop-nightly-2022-09-12
+
+MINIO_IMAGE=minio/minio:latest
+MINIO_MC_IMAGE=minio/mc
+MONGODB_IMAGE=mongo:5.0.12
+ORTHANC_IMAGE=osimis/orthanc:22.9.0
+RABBITMQ_IMAGE=rabbitmq:3.10-management
+
+
+# Default Listening AE Title
+INFORMATICS_GATEWAY_AE_TITLE=MONAI-DEPLOY
+# Service ports
+# Informatics Gatewway Ports - Changing the following values requires update to informatics-gateway.json
+INFORMATICS_GATEWAY_SCP_PORT=104
+INFORMATICS_GATEWAY_HL7_PORT=2575
+INFORMATICS_GATEWAY_API_PORT=5000
+# End Informatics Gatewway Ports
+
+WORKFLOW_MANAGER_PORT=5001
+TASK_MANAGER_PORT=5002
+
+MINIO_API_PORT=9000
+MINIO_CONSOLE_PORT=9001
+MONGODB_PORT=27017
+
+# Orthanc Ports- Changing the following values requires update to orthanc.json
```

Comment:
Please verify the comment or make it clear, as the statement is not right `Changing the following values requires update to orthanc.json`

I have had an Orthanc instance installed and running on my host, and the default Orthanc ports are already used. So I had to update the .env file to use `14242` and `18042` respectively. However, if these port values are also used in the configs/orthanc.json, Orthanc docker would not work at all. Also per docker-compose.yml, the Orthanc defaults ports are used within docker network, so there is no need to change the orthanc.json for docker to docker comm.

E,g, I changed the Orthanc UI port to 18042, `http://localhost:18042/app/explorer.html` while the config/orthanc.json still has the defautls.

Reply:
Removed